frontend/sportsmatch-app/src/components/HostEventComponent.tsx

```diff
@@ -139,6 +139,22 @@ function HostEventComponent() {
     toggle()
   }
 
+  // retrieving users rank
```

Comment:
this was removed from here in SMA-106 because isUserInRank is not parameter of JoinEventComponent anymore (JEC now handles retrieving user's rank)